Tests/functional/TransformationSandboxTest.php

```diff
@@ -0,0 +1,259 @@
+        $configId = $this->components->addConfiguration($configuration)['id'];
+        $configuration->setConfigurationId($configId);
+        $configurationRow = new ConfigurationRow($configuration);
+        $configurationRow->setConfiguration($data);
+        $configurationRow->setName('test-row');
+        $rowId = $this->components->addConfigurationRow($configurationRow)['id'];
+        return ['configId' => $configId, 'rowId' => $rowId];
+    }
+
+    public function testTables(): void
+    {
+        $configuration = [
+            'backend' => 'docker',
+            'description' => 'Test configuration',
+            'type' => 'r',
+            'input' => [
+                [
+                    'source' => 'in.c-main.source',
+                    'destination' => 'destination.csv',
+                    'whereColumn' => '',
+                    'changedSince' => '',
+                ],
+                [
+                    'source' => 'in.c-main.source',
+                    'destination' => 'filtered.csv',
+                    'whereColumn' => 'id',
+                    'whereOperator' => 'eq',
+                    'whereValues' => ['68640847'],
+                    'changedSince' => '',
+                ],
+            ],
+            'output' => [
+                [
+                    'destination' => 'out.c-main.r-transpose',
+                    'source' => 'transpose.csv',
+                ],
+            ],
+            'packages' => [],
+            'tags' => [],
+            'queries' => [
+                'this is some script\ncode on multiple lines',
+            ],
+        ];
+
+        $vars = $this->createConfiguration($configuration);
+        $envs = [
+            'KBC_TOKEN' => getenv('KBC_TEST_TOKEN'),
+            'KBC_STORAGEAPI_URL' => getenv('KBC_TEST_URL'),
+            'KBC_CONFIG_ID' => $vars['configId'],
+            'KBC_ROW_ID' => $vars['rowId'],
+            'KBC_CONFIG_VERSION' => '2',
+        ];
+        $specification = new DatadirTestSpecification(
+            __DIR__ . '/transformation-sandbox-tables/source/data',
+            0,
+            null,
+            '',
+            __DIR__ . '/transformation-sandbox-tables/expected/data/in'
+        );
+        $tempDatadir = $this->getTempDatadir($specification);
+        $process = $this->runScript($tempDatadir->getTmpFolder(), $envs);
+        $this->assertMatchesSpecification($specification, $process, $tempDatadir->getTmpFolder());
+        $output = $process->getOutput();
+        $output = preg_replace('#\[.*?\]#', '', $output);
+        $output = preg_replace('#\{.*?\}#', '', $output);
+        self::assertEquals(
+            implode("\n", [
+                ' app-logger.INFO: Starting Data Loader  ',
+                ' app-logger.INFO: DataLoader is loading data  ',
+                ' app-logger.INFO: Reading configuration ' . $vars['configId'] . ', row: ' . $vars['rowId'] . '  ',
+                ' app-logger.INFO: Script size 43  ',
```

Comment:
This is a strange message for me, not immediately sure what it's telling me, feels like debug.  Will leave comment at the source.